src/common/legacy_config_opts.h

```diff
@@ -412,6 +412,7 @@ OPTION(mds_log_max_events, OPT_INT)
 OPTION(mds_log_events_per_segment, OPT_INT)
 OPTION(mds_log_segment_size, OPT_INT)  // segment size for mds log, default to default file_layout_t
 OPTION(mds_log_max_segments, OPT_U32)
+OPTION(mds_log_max_expiring_segments, OPT_U32)
```

Comment:
We don't add options to this file anymore. Please remove this change.

Reply:
oh... sorry